src/js/components/Pagination/PageIndex.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+export const PageIndex = ({ page, ...rest }) => {
+  return (
+    <Box as="li">
+      {page === 'more-prev' || page === 'more-next' ? (
+        <Box pad="small">
```

Comment:
To keep the DOM simpler, could we remove the Box and just do `<Text margin="small" weight="bold">&#8230;</Text>`?

Reply:
Agreed. Adjusted.